improver_tests/acceptance/SHA256SUMS

```diff
@@ -691,32 +691,32 @@ a723b5cc5a2ba050a33dcc569ba67b93e8043d50bbd868cfcd065d000ae52b39  ./weighted_ble
-09d90a5e08da2546c2022f20d2429acc3d5e40d99f66ea6829b78308b388b6ee  ./wet-bulb-temperature/global/pressure_input.nc
-c100701d272fa226782f9aae296107cc58dfc02cc62cbdf50ebc60007922a787  ./wet-bulb-temperature/global/relative_humidity_input.nc
-d9401dfaf29e1b3d8a5cd84695613d7a7ee0d6ac62c7d157aad1908980f6c122  ./wet-bulb-temperature/global/temperature_input.nc
-bebdae181467d87e770ecd4603fd9b8a162a5c3c86cd5885ebe61c9042adbe3e  ./wet-bulb-temperature/multi_level/enukx_multilevel_pressure.nc
-c44adba807281615d06e282172fac379fc65cc545deb4841222c9adee865d75e  ./wet-bulb-temperature/multi_level/enukx_multilevel_relative_humidity.nc
-c34e4efce804613ffbd8104a36bdefd33432347de66f83494b059ee2b2e8619d  ./wet-bulb-temperature/multi_level/enukx_multilevel_temperature.nc
-0456be85918fd355f02fac1c1bb2568ad7e7907524dd4207b8d1556376215ed0  ./wet-bulb-temperature/multi_level/kgo.nc
+ef62d78071045163858387292ec56c11aa92e1fa6f5580c1e4ac485cd01979fc  ./wet-bulb-temperature-integral/basic/input.nc
+4143e7583d66c6e776e43e619622eab00fdee3293becadfaaa00ddd72147ac6a  ./wet-bulb-temperature-integral/basic/with_id_attr/kgo.nc
+f854f85ef58e2a0dc41c3bcdf940a23daa2782a50d3eaf798335fefa8bea21b8  ./wet-bulb-temperature-integral/basic/without_id_attr/kgo.nc
+ed4e971a00d56d771d79ad5dec90d7be8aa1dafc19811df5734c0d602e5ec57b  ./wet-bulb-temperature-integral/realizations/input.nc
+df4703b413617aff90adc15d351ac6e4488d70a570c58e2a7979dcd2459256f6  ./wet-bulb-temperature-integral/realizations/kgo.nc
+64396013800622079dd08b20b07274aeb73232220baf135738f6861bdd306f7c  ./wet-bulb-temperature/basic/enukx_pressure.nc
+634d52800e5783cd021f7af31809263e3345197e9c63184234d8348b1d637cd3  ./wet-bulb-temperature/basic/enukx_relative_humidity.nc
+2575b46b1d57dd37b3da85f7546c7cb5cc7b7637bb1a03cbee0b772b9a51c4b2  ./wet-bulb-temperature/basic/enukx_temperature.nc
+c67228e12dbb2c8f2008a47158b774953fe697f18c3eb7f6e585ad7c5c4d7b98  ./wet-bulb-temperature/basic/with_id_attr/kgo.nc
+9bdde50b0eabaf0c00f6cefee6e80a54e56fc1a1f4c55579e7352e2d0bb84c89  ./wet-bulb-temperature/basic/without_id_attr/kgo.nc
+4c27dcf1f4953f2eff731fbcc87897f65ecd018cfeac87b1525aeb2821bfe480  ./wet-bulb-temperature/global/kgo.nc
+e52d6a748419b57010810912e3933145d4823fba5c221e68e7af703d8b9e07fb  ./wet-bulb-temperature/global/pressure_input.nc
+637541033693a1ee55e73020a2b48219d340ffbbad59b9dbec531fa4ada24b16  ./wet-bulb-temperature/global/relative_humidity_input.nc
+29fd0062ca428745e6e00c0684bcd8ecde7c0883575fd6bb90afd81ef908912e  ./wet-bulb-temperature/global/temperature_input.nc
+e92e79b427929246fa669ef53327572b49ea005bdf24257c7dadca65dcab5caf  ./wet-bulb-temperature/multi_level/enukx_multilevel_pressure.nc
+9cc3ef17d4b48caf58b3faa808b2842bacc54673d1db5ddd7d800f943d811957  ./wet-bulb-temperature/multi_level/enukx_multilevel_relative_humidity.nc
+261af68b0dc16ae614f729d0cc29ecdc5789d215ee353aeb7fc03767a470e86e  ./wet-bulb-temperature/multi_level/enukx_multilevel_temperature.nc
```

Comment:
Is it worth considering thinning these fields along the height coordinate, or will this be considered later?

Reply:
I have thinned these vertically as it has no impact on the result of the calculation.